src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -380,6 +380,10 @@ export default defineComponent({
       return this.$store.getters.getExternalPlayer
     },
 
+    externalPlayerIsDefaultViewingMode: function () {
+      return this.externalPlayer !== '' && this.$store.getters.getDefaultViewingMode === 'external_player'
```

Comment:
Even if the setting is not possible/hidden on the settings page, it is still possible that the setting could have an incompatible value for the current install as people could copy their databases between installs.
```suggestion
      return process.env.IS_ELECTRON && this.externalPlayer !== '' && this.$store.getters.getDefaultViewingMode === 'external_player'
```